composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "glpi/glpi",
     "description": "GLPI dependencies",
-    "license": "GPL-2.0",
+    "license": "GPL-2.0-only",
```

Comment:
I do not agree, see https://github.com/glpi-project/glpi/blob/9.4/bugfixes/inc/commonglpi.class.php#L20 : «  either version 2 of the License, or (at your option) any later version. »

Reply:
indeed